tools/content-generator/src/main/java/org/projectnessie/tools/contentgenerator/cli/GenerateContent.java

```diff
@@ -130,98 +128,102 @@ public void execute() throws BaseNessieClientServerException {
-              tableName, commitToBranch.getName(), commitToBranch.getHash());
-
-      CommitMultipleOperationsBuilder commit =
-          api.commitMultipleOperations()
-              .branch(commitToBranch)
-              .commitMeta(
-                  CommitMeta.builder()
-                      .message(
-                          String.format(
-                              "%s table %s on %s, commit #%d of %d",
-                              tableContents != null ? "Update" : "Create",
-                              tableName,
-                              branchName,
-                              i,
-                              numCommits))
-                      .author(System.getProperty("user.name"))
-                      .authorTime(Instant.now())
-                      .build());
-      if (newContents instanceof IcebergTable || newContents instanceof IcebergView) {
-        commit.operation(Put.of(tableName, newContents, tableContents));
-      } else {
-        commit.operation(Put.of(tableName, newContents));
-      }
-      Branch newHead = commit.commit();
-
-      if (random.nextDouble() < newTagProbability) {
-        Tag tag = Tag.of("new-tag-" + random.nextLong(), newHead.getHash());
         spec.commandLine()
             .getOut()
             .printf(
-                "Creating tag '%s' from '%s' at %s%n", tag.getName(), branchName, tag.getHash());
-        api.createReference().reference(tag).sourceRefName(branchName).create();
-      }
+                "Committing content-key '%s' to branch '%s' at %s%n",
+                tableName, commitToBranch.getName(), commitToBranch.getHash());
 
-      try {
-        TimeUnit.NANOSECONDS.sleep(perCommitDuration.toNanos());
-      } catch (InterruptedException e) {
-        Thread.currentThread().interrupt();
-        break;
+        CommitMultipleOperationsBuilder commit =
+            api.commitMultipleOperations()
+                .branch(commitToBranch)
+                .commitMeta(
+                    CommitMeta.builder()
+                        .message(
+                            String.format(
+                                "%s table %s on %s, commit #%d of %d",
+                                tableContents != null ? "Update" : "Create",
+                                tableName,
+                                branchName,
+                                i,
+                                numCommits))
+                        .author(System.getProperty("user.name"))
+                        .authorTime(Instant.now())
+                        .build());
+        if (newContents instanceof IcebergTable || newContents instanceof IcebergView) {
+          commit.operation(Put.of(tableName, newContents, tableContents));
+        } else {
+          commit.operation(Put.of(tableName, newContents));
+        }
+        Branch newHead = commit.commit();
+
+        if (random.nextDouble() < newTagProbability) {
```

Comment:
again, why the randomness? Is it for performance reasons to not do it too often?